packages/cmrjs/cmr.js

```diff
@@ -221,17 +222,17 @@ class CMR {
    * Return object containing CMR request headers
    *
    * @param {string} [token] - CMR request token
-   * @param {boolean} isUMMG - boolean to determine content type of headers.
+   * @param {string} ummgVersion - UMMG metadata version string or '' if echo10 metadata
    * @returns {Object} CMR headers object
    */
-  getHeaders(token = null, isUMMG = false) {
-    const contentType = isUMMG ? 'application/vnd.nasa.cmr.umm+json;version=1.4' : 'application/echo10+xml';
+  getHeaders(token = null, ummgVersion = '') {
```

Comment:
Why '' instead of null?

Reply:
It just feels like null here would be more standard

Reply:
It might have been from when I was testing it somewhere else. But i can set it to null as a default value

Reply:
b12bf2a